spec/deaf_grandma_spec.rb

```diff
@@ -5,14 +5,16 @@
   let(:script) { DeafGrandma.new }
 
   it "says 'SPEAK UP SONNY!' when we speak regularly" do
-    expect(script.speak("Hi Grandma")).to eq "SPEAK UP SONNY!"
+    expect(script.speak("Hi Grandma")).to eq [true, "SPEAK UP SONNY!"]
```

Comment:
Changing this spec kinda feels like cheating, but I like how your implementation separates the logic to keep the  `speak` method almost pure and let the run loop deal with printing and exiting. This is my favorite version of deaf grandma I've seen so far.